packages/figma-utils/src/variables/parse.ts

```diff
@@ -108,14 +109,19 @@ export const resolveFigmaVariableValue = (
   value: VariableValue,
   allVariables: Record<string, Variable>,
   remBase: ParseFigmaVariablesOptions["remBase"] = 16,
+  name?: string,
 ): string => {
   if (typeof value === "number") {
+    if (name?.includes("font-weight")) return `${value}`;
     // numeric value, parse as rem or pixel value
     // note: value 0 should also be parsed as "0rem" instead of just "0" because otherwise
     // the CSS variable could not be used together with "calc()"
     if (remBase === false || remBase <= 0) return `${value}px`;
     return `${value / remBase}rem`;
   }
+  if (typeof value === "string") {
+    return `'${value}'`;
```

Comment:
```suggestion
    return `"${value}"`;
```

The resulting file uses double quotes so we should do the same here :)